pallets/api/src/messaging/test_utils.rs

```diff
@@ -26,10 +26,10 @@ pub fn ismp_post_request() -> PostRequest {
 		source: StateMachine::Polkadot(2000),
 		dest: StateMachine::Polkadot(2001),
 		nonce: 100u64,
-		from: vec![],
-		to: vec![],
+		from: [1u8].repeat(1000),
+		to: [1u8].repeat(1000),
```

Comment:
hmm, it seems like these are "module identifiers". Without looking into ISMP too closely, it seems likely the max size of this would be around 32 bytes (account id). 

These module IDs are controlled by the runtime, right? So, that would mean it's safe to set a max to 32 bytes.